src/__tests__/App.test.tsx

```diff
@@ -1,11 +1,13 @@
-import { shallow } from 'enzyme'
+import { render } from '@testing-library/react'
 import React from 'react'
 import { Provider } from 'react-redux'
 import configureStore from 'redux-mock-store'
 
 import App from '../App'
 
-it('renders without crashing', () => {
+// TODO: Causing Cannot log after tests are done. "Did you forget to wait for something async in your test? Attempted to log "Error: connect ECONNREFUSED 127.0.0.1:80 at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1133:16)"
+// ? MSW integration with tests
```

Comment:
TODO comment without a corresponding task => please add to https://github.com/kcdraidgroup/hospitalrun-frontend/projects/1

Reply:
Good protocol!